ReferralPlease/ReferralPlease/HomePageViewController.swift

```diff
@@ -26,6 +28,8 @@ class HomePageViewController: UIViewController, UITableViewDataSource {
         db = Firestore.firestore()
         
         tableView.dataSource = self
+
+        MentorRequests.update("id")
```

Comment:
@justinlim1 Is this PR finished and ready to merge? Not sure what this line means.

Reply:
yeah merge it. This line is just a place holder to test that class until the user id is properly passed to the vc

Reply:
You can use `User.currentUser()` to get the current logged in user. I not sure how to test this without seeing something working.